db/migrate/20170324171408_initial_migration.rb

```diff
@@ -8,6 +8,7 @@ def change
 
     create_table :resources do |t|
       t.string :name, null: false
+      t.string :description
```

Comment:
I realize that, I just haven't run any yet, so I figured it would be OK to update the migration.